community/kernel/src/main/java/org/neo4j/kernel/impl/transaction/log/pruning/ThresholdConfigParser.java

```diff
@@ -50,14 +50,16 @@ public static ThresholdConfigValue parse( String configValue )
         {
             switch ( boolOrNumber )
             {
-                case "true":
-                    return ThresholdConfigValue.NO_PRUNING;
-                case "false":
-                    return new ThresholdConfigValue( "entries", 1 );
-                default:
-                    throw new IllegalArgumentException( "Invalid log pruning configuration value '" + configValue +
-                            "'. The form is 'all' or '<number><unit> <type>' for example '100k txs' " +
-                            "for the latest 100 000 transactions" );
+            case "keep_all":
+            case "true":
+                return ThresholdConfigValue.NO_PRUNING;
+            case "keep_none":
+            case "false":
```

Comment:
Oh.. now I get it, it is a shared configuration. It makes sense to be retro compatible then.